### ISSUE TITLE
Window switcher shadow persists until overdrawn, after latest updates

### ISSUE THREAD

**Issue by activey** [elementary/gala#1443]
### What Happened?

I've found two issues:

1. When you have at least two windows open and there is a desktop background visible between them and you hit alt-tab then some strange artifacts are present after switcher disappears, please see the video attached,
2. I quite often make use of keyboard shortcuts for tiling windows either to left or right. When you use alt-tab this function stops working until you change focus to other window or display multitasking view.

### Steps to Reproduce

1. Open at least two windows
2. Make sure there is a space between so you can see the desktop background
3. Hit alt-tab and observe strange shadow artifacts on the screen
4. Now you can't also tile windows with predefined shortcuts until you change window focus by hand

https://user-images.githubusercontent.com/1751891/177555670-31105413-80f6-417c-84dc-0fea0817b011.mp4

### Expected Behavior

No artifacts should be present and window tile shortcuts should work as expected after using alt-tab.

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

**Comment by ded-ared** [comment]
As for the first point regarding strange artifacts, I guess it could be related somehow to [this issue](https://github.com/elementary/os-patches/issues/80). It still persists in eOS 6.1 (and it's very irritating). Waiting for OS 7.

**Comment by fayjie92** [comment]
@activey  You can use https://github.com/small-tech/catts
This solves this issue. This artifact is a part of Gala Alt+Tab and does not seem to be fixed soon. HAHA.

**Comment by jeremypw** [comment]
The shadow also persists when moving a PIP window around.

**Comment by activey** [comment]
@janxkoci yeah, shorcuts are back when you for example move the window by "hand", but still, this is way annoying :P

**Comment by janxkoci** [comment]
@activey alright, although I prefer to press some key rather than reach for a mouse...

**Comment by JSN-1** [comment]
i have the same issue with gala in fedora 36.

is there some way to remove drop shadow in gala ?

**Comment by activey** [comment]
@arfayjie it probably wont happen soon as there are more important matters like selecting folders with a single click: https://blog.elementary.io/updates-for-november-2022/

**Comment by arfayjie** [comment]
@activey  this is an annoying bug. Hard to use elementary if such bugs persist. It’s been more than 6 months. And it is the same for OS 7. How elementary team can ignore such bugs which should get the maximum attention. They can atleast ship catts until they fix it.

**Comment by activey** [comment]
@arfayjie tell me about it...

**Comment by arfayjie** [comment]
@peteruithoven, you have misunderstood my comment then. And also this is not a question to ask myself, what have I contributed? I have more contributions towards research than an operating system. Elementary OS claims to be user friendly mac and windows alternative. And there are some minimum requirements for a user to use it. It’s really an important issue. But I also provided the solution. Catts plugin doesn’t create this issue at all. Elementary os can ship it by default. That was what I meant. I understand people work on free time. So, why do we need to reinvent the wheel? If we already have a working solution.

**Comment by kgrubb** [comment]
> Elementary os can ship it by default. That was what I meant. I understand people work on free time. So, why do we need to reinvent the wheel? If we already have a working solution.

@arfayjie  Great idea! In fact, it already has been done: https://github.com/small-tech/catts#catts-is-now-part-of-elementary-os. 

Unfortunately, this issue is a bit more deeper, and hasn't been fully chased down yet. I agree that the issue is frustrating, but we don't have much to gain by complaining that something takes a long time to resolve. Additional logs, recordings, and screenshots would be much more helpful and constructive. That way we can further diagnose and hopefully resolve the issue!

**Comment by arfayjie** [comment]
@kgrubb  I got your point. That’s true. Actually I was trying to give some input here. I didn’t check the logs yet. But this is what I found so far.

If you use [catts](https://github.com/small-tech/catts#catts-is-now-part-of-elementary-os) and install it as gala plugin then you don’t see the artifacts. So what I can conclude is that the way it is implemented in elementary os is the starting point for debugging. 

May be @davidmhewitt @danirabbit would know it better. I will add some screenshots (or may be screen recording soon).
